lib/ansible/modules/database/postgresql/postgresql_set.py

```diff
@@ -104,7 +104,7 @@
     msg: "{{ set.name }} {{ set.prev_val_pretty }} >> {{ set.value_pretty }} restart_req: {{ set.restart_required }}"
   when: set.changed
 # Ensure that the restart of PostgreSQL server must be required for some parameters.
-# In this situation you see the same parameter in prev_val and value_prettyue, but 'changed=True'
+# In this situation you see the same parameter in prev_value and value_pretty, but 'changed=True'
```

Comment:
```suggestion
# In this situation you see the same parameter in prev_value_pretty and value_pretty, but 'changed=True'
```
it would be more correct:)

Reply:
prev_val_pretty is correct value as per code.